devel/libhttpd/Portfile

```diff
@@ -1,32 +1,37 @@
-PortSystem 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-name		libhttpd
-version		1.3
-categories	devel www
-license		GPL-2
-maintainers	nomaintainer
-platforms	darwin freebsd
-description	provides basic web server capabilities
+PortSystem          1.0
+
+name		        libhttpd
+version		        2.0
+revision            0
+categories	        devel www
+license		        GPL-2
+maintainers	        nomaintainer
+description	        provides basic web server capabilities
 long_description	libhttpd can be used to add basic web server \
-			capabilities to an application or embedded device. The \
-			library handles both static and dynamically generated \
-			content, has very low overheads, and provides many \
-			features to simplify the creation of web based \
-			application interfaces.
+			        capabilities to an application or embedded device. \
+			        The library handles both static and dynamically generated \
+			        content, has very low overheads, and provides many \
+			        features to simplify the creation of web based \
+			        application interfaces.
 
-homepage	http://www.hughes.com.au/products/libhttpd/
-master_sites	${homepage}
-checksums	md5 ea0fb523fdb467702d34185be55e940c
+homepage	        https://hughestech.com.au/
+master_sites	    ${homepage}downloads/${name}/
+distname            ${name}-${version}
+checksums           md5     0dbfb1218296c22ffa8c2731f667b967 \
```

Comment:
the `md5` checksum is obsolete and can be removed

Reply:
Macports suggested checksums line, I copied that. Will remove it.

Reply:
yeah, that's true: MacPorts will give you the udpated `md5` checksum if that was already present in the Portfile. The `md5` is pretty old and not actively used anymore, but might indeed still be present for some ports that haven't been updated lately - the suggestion is to remove it and only keep the three other ones.